scripts/auto-rebase/lvms_assets.yaml

```diff
@@ -31,5 +32,3 @@ assets:
         ignore: "provided by MicroShift"
       - file: topolvm_default-storage-class.yaml
         ignore: "provided by MicroShift"
-      - file: topolvm-configmap_lvms-version.yaml
-        ignore: "provided by MicroShift"
```

Comment:
This change is surprising. Why don't we need this any more?

Reply:
this was introduced earlier and is a config map generated in the script. Its still created and maintained, but since the manifest extraction does it it should not be ignored. (otherwise the file would not be copied from the staging repository).

Reply:
Here it is, now generated: https://github.com/openshift/microshift/pull/2993/files#diff-645a8be2d813fd3ee4d2d90e0377d488623dd8c77948101fe58a42a0af563bf0R7